src/features/query-bar/__tests__/__snapshots__/Condition-test.js.snap

```diff
@@ -105,7 +95,30 @@ exports[`features/query-bar/components/filter/Condition render() should render C
       updateSelectedField={[Function]}
       updateValueField={[Function]}
       valueKey={null}
-      valueOptions={Array []}
+      valueOptions={
+        Array [
```

Comment:
I'd like to see the minimum number of options to demonstrate the expected behavior. Are 4 options necessary here?

Reply:
I can cut it down to just one option